datafusion/sqllogictest/test_files/expr/date_part.slt

```diff
@@ -876,3 +1070,5 @@ true
 
 query error DataFusion error: Internal error: unit Nanosecond not supported
 SELECT (date_part('nanosecond', now()) = EXTRACT(nanosecond FROM now()))
+
+
```

Comment:
git encourages against trailing empty lines
We could add file ending normalization to slt --complete mode.